src/mlnet/Templates/Console/ModelProject.cs

```diff
@@ -27,15 +27,15 @@ public virtual string TransformText()
         {
             this.Write("<Project Sdk=\"Microsoft.NET.Sdk\">\r\n\r\n  <PropertyGroup>\r\n    <TargetFramework>netc" +
                     "oreapp2.1</TargetFramework>\r\n  </PropertyGroup>\r\n  <ItemGroup>\r\n    <PackageRefe" +
-                    "rence Include=\"Microsoft.ML\" Version=\"1.2.0\" />\r\n");
+                    "rence Include=\"Microsoft.ML\" Version=\"1.3.1\" />\r\n");
```

Comment:
ugh.  can you make it a const somewhere so it's not duplicated all over the place?